browser/help/topics/v3-ht-annotations.md

```diff
@@ -31,8 +33,22 @@ The gnomAD v3 Hail Table annotations are defined below:
   - **bin_freq**: Bin frequencies for the age histogram. This is the number of records found in each bin.
   - **n_smaller**: Count of age values falling below lowest histogram bin edge.
   - **n_larger**: Count of age values falling above highest histogram bin edge.
-- **freq_sample_count**: A sample count per sample grouping defined in the 'freq_meta' global annotation.
-
+- **freq_meta_sample_count**: A sample count per sample grouping defined in the 'freq_meta' global annotation.
+- **vrs_version**: The Variant Representation Specification that was used to compute IDs on the callset.
+  - **vrs_python_version**: The [version](https://github.com/ga4gh/vrs-python/tags) of the vrs-python library that was used to compute IDs on the callset.
+  - **seqrepo_version**: The [version](https://github.com/biocommons/biocommons.seqrepo) of the SeqRepo database that was used in VRS computations.
+- **tool_versions**: Versions of in silico predictors used in the callset.
+  - **dbsnp_version**: dbSNP version used in the callset.
```

Comment:
minor note that we included the full names of revel, cadd, sift, and polyphen, so it might also make sense to expand dbsnp as well

Reply:
I actually saw you removed some full names. I don't think we need to get full name for dbSNP, because it's very common in the community.

Reply:
it is common, but so is CADD, so maybe we should stop adding full names entirely or add full names to all.

also, I removed full names from the descriptions below because they are now spelled out here